components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -0,0 +1,45 @@
+use tidb_query_codegen::rpn_fn;
+
+use crate::codec::data_type::Bytes;
+use crate::error::Result;
+
+#[rpn_fn]
+#[inline]
+pub fn length(arg: &Option<Bytes>) -> Result<Option<i64>> {
+    Ok(match arg {
+        Some(s) => Some(s.len() as i64),
+        _ => None,
+    })
+}
+
+#[cfg(test)]
+mod tests {
+    use tipb::ScalarFuncSig;
+
+    use super::*;
+    use crate::rpn_expr::types::test_util::RpnFnScalarEvaluator;
+
+    #[test]
+    fn test_length() {
+        let test_cases = vec![
+            (ScalarFuncSig::Length, "", Some(0i64)),
+            (ScalarFuncSig::Length, "你好", Some(6i64)),
+            (ScalarFuncSig::Length, "TiKV", Some(4i64)),
+            (ScalarFuncSig::Length, "あなたのことが好きです", Some(33i64)),
+            (ScalarFuncSig::Length, "분산 데이터베이스", Some(25i64)),
+            (ScalarFuncSig::Length, "россия в мире  кубок", Some(38i64)),
```

Comment:
```sql
mysql iosmanthus@(none):test> select length("россия в мире  кубок")                                                               
+--------------------------------+
| length("россия в мире  кубок") |
+--------------------------------+
| 36                             |
+--------------------------------+
1 row in set
Time: 0.024s
```

Reply:
Looks like this is either a GitHub problem or browser problem.. It is fine when you copy the string from local file.